s2spy/rgdr/rgdr.py

```diff
@@ -508,8 +508,9 @@ def fit(self, precursor: xr.DataArray, timeseries: xr.DataArray):
         masked_data = masked_spherical_dbscan(
             precursor, corr, p_val, self._dbscan_params
         )
-
-        self._clusters = masked_data.cluster_labels
+        self.corr_map = corr
+        self.pval_map = corr
```

Comment:
New public API attributes should be documented.

Reply:
Thanks for the comments Peter! 

How do I update the API documentation? I was kind of assuming that this would go automatically.

Reply:
See e.g. https://stackoverflow.com/a/3051356

Reply:
The attributes are documented in the docstrings now.